src/stylesheets/project_specific/developer/_developer-settings.scss

```diff
@@ -4,4 +4,19 @@
     margin-right: 12px;
   }
 
+  .plan .free {
```

Comment:
I'm generally really antsy with class names like `.plan .free`. Even though it's nested here for this page, someone else might come along and make a top-level component class `.plan` and suddenly it collides with the one here.

Also, this only `@extend`s one class name, and it's very much related to the toggle switch, I would imagine that it's better to alter the markup to use the `.toggle-switch-text-disabled/enabled` class names directly?